main/tests/cypress/cypress/integration/create-project/create_project.spec.js

```diff
@@ -26,89 +26,168 @@ describe(__filename, function () {
-    const uniqueProjectName = Date.now();
-    const uniqueTagName1 = 'tag1_' + Date.now();
-    const uniqueTagName2 = 'tag2_' + Date.now();
 
+    // preview and click next
+    cy.doCreateProjectThroughUserInterface();
+  });
+  it('Test the create project from clipboard based on TSV', function () {
+    // navigate to the create page
+    cy.visitOpenRefine();
+    cy.navigateTo('Create Project');
+    cy.get('#create-project-ui-source-selection-tabs > div')
+      .contains('Clipboard')
+      .click();
+    cy.get('#or-import-clipboard').should(
+      'to.contain',
+      'Paste data from clipboard here:'
+    );
+    // add file
+    const tsvFile = `Some parameter	Other parameter	Last parameter
+    CONST	123456	12.45`;
+    cy.get('textarea').invoke('val', tsvFile);
+    cy.get(
+      '.create-project-ui-source-selection-tab-body.selected button.button-primary'
+    )
+      .contains('Next »')
+      .click();
     cy.get(
       '.default-importing-wizard-header input[bind="projectNameInput"]'
-    ).type(uniqueProjectName);
-    // triger the select input
-    cy.get('#project-tags-container').click();
-    // Type and Validate the tag, pressing enter
-    cy.get('#project-tags-container .select2-input').type(uniqueTagName1);
-    cy.get('body').type('{enter}');
-    cy.get('#project-tags-container .select2-input').type(uniqueTagName2);
-    cy.get('body').type('{enter}');
-    cy.get('#or-import-parsopt').click();
-    cy.doCreateProjectThroughUserInterface();
+    ).should('have.value', 'Clipboard');
 
-    cy.visitOpenRefine();
-    cy.navigateTo('Open Project');
-    cy.get('#projects-list')
-      .contains(uniqueProjectName)
-      .parent()
-      .parent()
-      .contains(uniqueTagName1);
-    cy.get('#projects-list')
-      .contains(uniqueProjectName)
-      .parent()
-      .parent()
-      .contains(uniqueTagName2);
-  });
+    // then ensure we are on the preview page
+    cy.get('.create-project-ui-panel').contains('Configure Parsing Options');
 
-  it('E2E, Creates a simple project, based on a CSV', function () {
+    // preview and click next
+    cy.doCreateProjectThroughUserInterface();
+  });
+  it('Test the create project from Web URL based on CSV', function () {
     // navigate to the create page
     cy.visitOpenRefine();
     cy.navigateTo('Create Project');
+    cy.get('#create-project-ui-source-selection-tabs > div')
+      .contains('Web Addresses (URLs)')
+      .click();
+    cy.get('#or-import-enterurl').should(
+      'to.contain',
+      'Enter one or more web addresses (URLs) pointing to data to download:'
+    );
     // add file
-    const csvFile = {
-      filePath: 'food.mini.csv',
-      mimeType: 'application/csv',
-    };
-    cy.get(
-      '.create-project-ui-source-selection-tab-body.selected input[type="file"]'
-    ).attachFile(csvFile);
+    const csvURL =
+      'https://people.sc.fsu.edu/~jburkardt/data/csv/addresses.csv';
```

Comment:
@kushthedude what is this domain ?
I don't think it's a good idea to rely on external domains for ou test suite
Is it possible to find a workaround using files stored directly on this Git repository ? Or something owned by OpenRefine ?
Alternative: Possible to use a local url ? Using cypress mock capabilities  ?

Reply:
Let's not make it a local URL, since we need to test openrefine fetch and upload API. I agree to put csv in the openrefine but the csv which is attached is public license, we can copy its contents or either we can provide here the link to one of our cypress fixture.

Reply:
For the wikimedia extension, I am testing like that
```
 cy.get('.add-wikibase-dialog input[bind="manifestURLInput"]').type(
      'https://raw.githubusercontent.com/OpenRefine/wikibase-manifests/master/openrefine-wikibase-test-manifest.json'
    );
```
maybe you can use github

Reply:
Yes, there are lots of URLs that we control. Github, openrefine.org, etc. Let's pick one that we control.

Reply:
done!